src/core/fs.cc

```diff
@@ -1,4 +1,6 @@
 #include "core.hh"
+#include "uv.h"
+#include <sys/_types/_s_ifmt.h>
```

Comment:
@jwerle I forget, do we still need this?

Reply:
nope we can remove that

Reply:
Both of them?

Reply:
It's both of them.